content/sensu-go/5.19/reference/checks.md

```diff
@@ -1005,6 +1005,50 @@ spec:
 
 {{< /language-toggle >}}
 
+### PowerShell script in check commands
+
+If you use a PowerShell script in your check command, make sure to include the `-f` flag in the command.
```

Comment:
May want to off some explanation into why the `-f` is required. My understanding of it is that it allows the proper exit code to be passed into Sensu.

Reply:
Good point, thank you. I will add it.